### PR TITLE
Add input debug info

### PR DESCRIPTION
**Pull request by Kiisu-Master**

### CODE REVIEW

**Comment by MewPurPur** [comment]
Looks good! Some nitpicks:

- 10 lines is too much, I'd say keep 5 lines at most to give space for other things.
- Clear the label when it's hidden, start filling it in when it's shown by F3.

File: src/ui_parts/display.gd [hunk 1]
```diff
@@ -202,8 +204,28 @@ func update_debug() -> void:
...
+
+var last_input := ""
+var last_input_count := 1
+
+func _input_debug_handler(event: InputEvent) -> void:
+	if event.is_pressed():
+		var debug_text := input_debug_label.text
+		var event_text := event.as_text()
+		if event_text == last_input:
+			debug_text = debug_text.left(debug_text.left(-1).rfind("\n") + 1)
```
Comment (MewPurPur) [inline src/ui_parts/display.gd:222]:
```suggestion
			debug_text = debug_text.left(debug_text.rfind("\n", -2) + 1)
```

Comment (Kiisu-Master) [reply src/ui_parts/display.gd:222]: This doesn't work

Comment (MewPurPur) [reply src/ui_parts/display.gd:224]: Huh. Sorry then

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**